webapp/web/configuration.css

```diff
@@ -208,6 +208,7 @@
 
 .conversation-response-language__text--alert {
   border-color: #ff0000;
+  outline-color: #ff0000;
```

Comment:
single pixel was fine, this is now too bright. please revert here and below as well

Reply:
actually I see you're using it to show that the box is selected.. so let's keep the outline, but can we use a less red colour for both border and outline? maybe something like `salmon` or something closer to that rather than `#ff0000`?

Reply:
Changed also the counter to salmon